src/js/Keyboard.js

```diff
@@ -67,25 +68,77 @@ export default class Keyboard extends Component {
     });
   };
 
+  handleUserMask = () => {
+    const toggleMask = !this.state.userMaskedInput
+    this.setState({
+      userMaskedInput: toggleMask
+    })
+    const event = toggleMask ? "INPUT_KEY_MASK_ENABLED" : "INPUT_KEY_MASK_DISABLED"
+    uiController.onKeyboardInput(null, event);
+  }
+
+  replaceSpecialCharacters(keyboardLayout, characters) {
+    var layout = keyboardLayout.default;
+    var customCharacters = characters;
+    //Special characters on root level of supported keyboards
+    var replaceableChars = '`-=[]\\;\',./^+#<&(_)=$:!@';
```

Comment:
```suggestion
    var replaceableChars = '`-=[]\\;\',./^+#<&(_)$:!@';
```
Seems that `=` was included twice